server/sonar-server/src/main/java/org/sonar/server/computation/measure/MeasureRepositoryImpl.java

```diff
@@ -173,4 +237,55 @@ public String apply(@Nonnull BatchReport.Measure input) {
       return input.getMetricKey();
     }
   }
+
+  @Immutable
+  private static final class MeasureKey {
+    private static final int DEFAULT_INT_VALUE = -6253;
```

Comment:
two reasons:
* `int` uses less memory than `Integer`
* saves on handling the case where property is null (slighly faster)

minor optimization for an object which is used as a map key and which equals and hashcode methods are potentially highly used